### PR TITLE
[Deprecated, To Be Closed] Eslint + prettier

### PR DESCRIPTION
**Pull request by radik**
POC for eslint

### CODE REVIEW

File: src/basis/basis.js [hunk 1]
```diff
@@ -309,16 +341,19 @@ export class Basis {
      * This guarantees the independence from the order in the elements array.
      * @return {String}
      */
-    getAsSortedString(inStandardRepresentation = true) {
+    getAsSortedString() {
```
Comment (timurbazhirov) [inline src/basis/basis.js:344]: This is a potentially dangerous behavior: removing default function args ...

File: src/abstract/array_with_ids.js [hunk 2]
```diff
@@ -30,31 +31,38 @@ export class ArrayWithIds {
...
      */
     mapArrayInPlace(fn) {
-        if (!_.isFunction(fn)) throw new Error("ArrayWithIds.mapArray: must pass function as argument");
+        if (!_.isFunction(fn))
+            throw new Error('ArrayWithIds.mapArray: must pass function as argument');
         this.array = this.array.map((...args) => fn(...args));
     }
 
-    getArrayElementByIndex(idx) {return this.array[idx]}
+    getArrayElementByIndex(idx) {
```
Comment (timurbazhirov) [inline src/abstract/array_with_ids.js:39]: Maybe we can configure the linter to keep one-line functions?

File: src/abstract/array_with_ids.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import _ from "underscore";
+import _ from 'underscore';
 
-import {ScalarWithId} from "./scalar_with_id";
+import { ScalarWithId } from './scalar_with_id';
```
Comment (timurbazhirov) [inline src/abstract/array_with_ids.js:3]: Not sure about replacing all `"` with `'`. Pycharm/Webstorm by default put double-quotes for imports. Maybe we should just keep it that way?

File: src/lattice/reciprocal/paths.js [hunk 4]
```diff
@@ -1,486 +1,132 @@
...
-            "Г",
-            "R",
-            "X"
-        ],
-        [
-            "M",
-            "R"
-        ]
+    CUB: [
+        ['Г', 'X', 'M', 'Г', 'R', 'X'],
```
Comment (timurbazhirov) [inline src/lattice/reciprocal/paths.js:10]: The array elements were on newlines for a reason: easier to read and spot errors in each, especially for formulas, and we can move/add elements without affecting the rest of them. Can we keep it the way they were?

**Review by timurbazhirov — CHANGES_REQUESTED** [review]
We need to converge on eslint settings

File: src/basis/basis.js (hunk 1 above)
Comment (radik) [reply src/basis/basis.js:344]: It's not used.

File: src/abstract/array_with_ids.js (hunk 2 above)
Comment (radik) [reply src/abstract/array_with_ids.js:39]: We can change it to:
```
getArrayElementByIndex = (idx) => this.array[idx];
```

File: src/abstract/array_with_ids.js (hunk 3 above)
Comment (radik) [reply src/abstract/array_with_ids.js:3]: Ok, no problem

File: src/abstract/array_with_ids.js (hunk 2 above)
Comment (timurbazhirov) [reply src/abstract/array_with_ids.js:39]: Sounds good

File: src/lattice/reciprocal/symmetry_points.js [hunk 5]
```diff
@@ -1,1563 +1,878 @@
...
-                point: 'X',
-                coordinates: [
-                    1 / 2,
-                    0.0,
-                    1 / 2
-                ]
-            }
-        ]
+                point: "K",
+                coordinates: [3 / 8, 3 / 8, 3 / 4],
```
Comment (timurbazhirov) [inline src/lattice/reciprocal/symmetry_points.js:30]: Here having numbers on separate lines is actually more readable.

File: src/basis/basis.js (hunk 1 above)
Comment (timurbazhirov) [reply src/basis/basis.js:344]: OK

**Pull request closed without merging**